src/main/java/edu/harvard/iq/dataverse/FileDownloadServiceBean.java

```diff
@@ -291,10 +291,10 @@ public void downloadCitationRIS(FileMetadata fileMetadata, Dataset dataset) {
 
         }
     }
-    
-    private String getFileNameDOI(DatasetVersion workingVersion) {
+
+    private String  getFileNameDOI(DatasetVersion workingVersion) {
```

Comment:
Its just a space, but now that we are looking at it, do we use a formatting setting in eclipse?